README.md

```diff
@@ -122,553 +121,587 @@ around-the-clock shop with some breaks.
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## evaluation tool
+## Evaluation tool
 
-Please have a look at the [evaluation tool] which can give you an impression how this library can be used and what it is capable of.
+Please have a look at the [evaluation tool] which can give you an impression of how this library can be used and what it is capable of.
 
-A mirror is set up at https://openingh.ypid.de/evaluation_tool/
+A mirror is set up at <https://openingh.ypid.de/evaluation_tool/>
 
-## Install
+## Installation
 
 ### For Developer
+
 Just clone the repository:
 
-```Shell
+```text
 git clone --recursive https://github.com/opening-hours/opening_hours.js.git
 ```
```

Comment:
Clearly a shell command, same below :)

Reply:
`Shell` isn't a language supported by GitHub. Your options are `sh, shell-script, bash or zsh` https://github.com/github/linguist/blob/1c106773c7496fa23e1a2dd04f27aa9be4ddff3c/lib/linguist/languages.yml#L5194-L5202

Spoler alert they all look the same as text

text
```text
git clone --recursive https://github.com/opening-hours/opening_hours.js.git
```

sh
```sh
git clone --recursive https://github.com/opening-hours/opening_hours.js.git
```

For semantics sake, I'll make them `sh`.

Reply:
Fixed in db3ee9c